packages/ui/src/accounts/components/AccountItem/components/styles.ts

```diff
@@ -72,6 +72,7 @@ export const LocksButtons = styled.div`
   align-self: end;
   display: flex;
   column-gap: 10px;
+  align-items: center;
```

Comment:
this might be not needed, nothing change if i remove this style:
<img width="719" alt="image" src="https://user-images.githubusercontent.com/9252017/206583867-ff1558a4-5999-49b4-9c23-1760c596ab26.png">

Reply:
fixed